polyfill.d.ts

```diff
@@ -1,10 +1,13 @@
 declare global {
-    interface Array<T> {
-        with(index: number, value: T): T[];
-        toReversed(): T[];
-        toSorted(compareFn?: (a: T, b: T) => number): T[];
-        toSpliced(start: number, deleteCount?: number, ...values: T[]): T[];
-    }
+	interface Array<T> {
+		with(index: number, value: T): T[];
+		toReversed(): T[];
+		toSorted(compareFn?: (a: T, b: T) => number): T[];
+		toSpliced(start: number, deleteCount?: number, ...values: T[]): T[];

```

Comment:
btw there are also whitespace changes here. I'm happy to switch to tabs, but we should probably keep the whole file consistent 🙂